### DISCUSSION TITLE
Plex - No permanent access token found

### DISCUSSION THREAD

**Discussion by goldsworthy87** [arabcoders/watchstate#314] [General] [answered]
Trying to setup my plex server, but am getting the error below.

logger.ERROR: No permanent access token found in [plex] user [USERNAME] response. {"user_id":"USERID"} []

Am I doing something wrong?

**Comment by arabcoders** [comment, marked as answer]
> Trying to setup my plex server, but am getting the error below.
> 
> logger.ERROR: No permanent access token found in [plex] user [USERNAME] response. {"user_id":"USERID"} []
> 
> Am I doing something wrong?

Maybe Plex has changed how they report the tokens, you could try running the following `config:edit --add --trace -vvv --context -- backend_name` and follow the prompt. do not share the result as it contains sensitive information. there might be a problem with plex.tv API at the moment.

You could also try re-running the command again i think it was probably an issue with plex.tv API
